configs/components/rubygem-multi_json.rb

```diff
@@ -1,6 +1,6 @@
 component "rubygem-multi_json" do |pkg, settings, platform|
-  pkg.version '1.13.1'
```

Comment:
Unfortunately this is another gem that is shared by most/all projects. I usually have just raised a separate PR for these updates so that teams have time to review it without blocking what I want to get in.